TemplateExample.swift

```diff
@@ -0,0 +1,48 @@
+// Generated by Unstringify.
```

Comment:
I would move this file to the Examples directory.

Also I would change this line to:
`// Generated by Unstringify with a custom template`

Reply:
Done!